pkg/machines/components/networks/network.jsx

```diff
@@ -157,12 +164,11 @@ class NetworkActions extends React.Component {
                     {_("Activate")}
                 </Button>
                 }
-                <DeleteResource objectType="Network"
-                    objectId={id}
-                    objectName={network.name}
-                    deleteHandler={() => deleteHandler(network)}
-                    overlayText={_("Non-persistent network cannot be deleted. It ceases to exists when it's deactivated.")}
-                    disabled={!network.persistent} />
+                {this.state.deleteDialogProps && <DeleteResourceModal {...this.state.deleteDialogProps} />}
+                <DeleteResourceButton objectId={id}
```

Comment:
However, I don't understand how this is supposed to help -- the whole `<NetworkActions>` is still part of a `<ListingRow>`, so if the list gets re-rendered, then both the dialog and the button will be re-created?

Reply:
It's not making any difference. It's still part of they row in this specific file. But this Listing is the PF3 implementation which has not issues with this so I thought I could just let it there for now. I can move it now if you prefer. The other two dialogs were using PF4 lists which really caused an issue.

Reply:
Given that the test still failed once with the exact same symptom, I suppose the PF3 list has the same issue?